File: src/main/java/org/ghrobotics/robot/Robot.java
package org.ghrobotics.robot;

import com.ctre.phoenix.motorcontrol.ControlMode;
import com.ctre.phoenix.motorcontrol.can.TalonSRX;
import edu.wpi.first.wpilibj.GenericHID;
import edu.wpi.first.wpilibj.TimedRobot;
import edu.wpi.first.wpilibj.XboxController;
import edu.wpi.first.wpilibj.command.Scheduler;


public class Robot extends TimedRobot {

    @Override
    public void robotInit() {
        // Add your robot initialization code here
    }

    @Override
    public void robotPeriodic() {
        Scheduler.getInstance().run();
    }

    @Override
    public void teleopInit() {
        // Add your teleop initialization here
    }

    @Override
    public void teleopPeriodic() {
        // Add your teleop code here
    }

    @Override
    public void autonomousInit() {
        // Add your autonomous initialization here
    }

    @Override
    public void autonomousPeriodic() {
        // Add your autonomous code here
    }
}

File: src/main/kotlin/org/ghrobotics/frc2020/Robot.kt
/*
 * FRC Team 5190
 * Green Hope Falcons
 */

package org.ghrobotics.frc2020

import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard
import org.ghrobotics.frc2020.subsytems.Drivetrain
import org.ghrobotics.lib.wrappers.FalconTimedRobot

object Robot : FalconTimedRobot() {

    // Constructor of the Robot class.
    init {
        // Register the drivetrain into the SubsystemHandler.
        +Drivetrain
    }

    // Runs once when robot boots up
    override fun robotInit() {}

    // Runs once when autonomous period starts
    override fun autonomousInit() {}

    // Runs once when teleop period starts
    override fun teleopInit() {}

    // Runs once when robot is disabled
    override fun disabledInit() {}


    // Runs every 20 ms when robot is on
    override fun robotPeriodic() {
        Shuffleboard.update()
    }

    // Runs every 20 ms when autonomous is enabled
    override fun autonomousPeriodic() {}

    // Runs every 20 ms when teleop is enabled
    override fun teleopPeriodic() {}

    // Runs every 20 ms when robot is disabled
    override fun disabledPeriodic() {}
}